.circleci/config.yml

```diff
@@ -46,3 +46,18 @@ workflows:
           context: ipfs-dns-deploy
           requires:
             - build
+  # deploy the blog evey day at 00:00 UTC, to allow scheduled posting.
+  nightly:
```

Comment:
It would be ideal to have one workflow that can be triggered by either a push or a schedule, but it doesn't look like circle support that, hence the duplication

https://circleci.com/docs/2.0/configuration-reference/#triggers